packages/e2e-tests/specs/editor/blocks/__snapshots__/buttons.test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`Buttons can jump to the link editor using the keyboard shortcut 1`] = `
 "<!-- wp:buttons -->
 <div class=\\"wp-block-buttons\\"><!-- wp:button -->
-<div class=\\"wp-block-button\\"><a class=\\"wp-block-button__link\\">WordPress</a></div>
+<div class=\\"wp-block-button\\"><a class=\\"wp-block-button__link\\" href=\\"https://www.wordpress.org/\\" title=\\"https://www.wordpress.org/\\">WordPress</a></div>
```

Comment:
This change highlights another issue: I think we shouldn't want to assign a `title` attribute here? Based on what a `title` would be used for ([advisory information](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)), I would think it _less_ meaningful than the text it's supposedly meant to advise.

It also seems like it could be an accessibility concern, to the extent that I might question whether we want to be surfacing `title` at all from `LinkControl`.

References:

>Relying on the title attribute is currently discouraged as many user agents do not expose the attribute in an accessible manner as required by this specification (e.g., requiring a pointing device such as a mouse to cause a tooltip to appear, which excludes keyboard-only users and touch-only users, such as anyone with a modern phone or tablet).

https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute

Or: https://silktide.com/blog/2013/i-thought-title-text-improved-accessibility-i-was-wrong

Reply:
`title` should only be used to provide legitimate context to the link. It shouldn't repeat the link text. Therefore if the `title` isn't handcrafted by the user then it should be omitted.

Reply:
> `title` should only be used to provide legitimate context to the link. It shouldn't repeat the link text. Therefore if the `title` isn't handcrafted by the user then it should be omitted.

@getdave You might also be interested in my comment at https://github.com/WordPress/gutenberg/pull/19462#discussion_r366566720 , where I share a few more resources which would seem to support the claim that the `title` attribute should never be used for links.

>[[1]](https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute) [[2]](https://silktide.com/blog/2013/i-thought-title-text-improved-accessibility-i-was-wrong) [[3]](https://developer.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/) [[4]](https://www.24a11y.com/2017/the-trials-and-tribulations-of-the-title-attribute/) [[5]](https://a11yproject.com/posts/title-attributes/)

>https://core.trac.wordpress.org/ticket/24766

Do you have any more context for the value we were hoping to get out of it?